extensions/CodeBlocks/package.json

```
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "codeblocks",
+  "title": "CodeBlocks",
+  "description": "Set or modify the programming language in markdown code blocks in batch.",
+  "icon": "CodeBlocks.png",
+  "author": "isfeng",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Set Language",
+      "subtitle": "Code Blocks",
```

Sorry for the nitpicking, but just noticed this. In order to be coherent with the extension name, it should be CodeBlocks right?
```suggestion
      "subtitle": "CodeBlocks",
```

---

Yeah, thank you @danielsequeira